apm-checkstyle/checkStyle.xml

```diff
@@ -96,7 +96,7 @@
             <property name="format" value="(^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$)"/>
         </module>
         <module name="PackageName">
-            <property name="format" value="^org\.apache\.skywalking(\.[a-zA-Z][a-zA-Z0-9]*)+$"/>
+            <property name="format" value="^(org|test)\.apache\.skywalking(\.[a-zA-Z][a-zA-Z0-9]*)+$"/>
```

Comment:
Question, in spring 4.1.x test case, we have this package name, so why we change this now? Does only e2e follow this style file, but plugin tests not?

Reply:
@kezhenxu94 @dmsolr

Reply:
> Question, in spring 4.1.x test case, we have this package name, so why we change this now? Does only e2e follow this style file, but plugin tests not?

Yes obviously, there is checkstyle plugin in E2E `pom.xml`, but not in the plugin tests,

Reply:
@dmsolr Please take care of plugin test codes when you have time.

Reply:
PluginTests don't enable checkstyle yet. ref #4377